### DISCUSSION TITLE
part-ii/2008-97/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#75] [Question comments]
Part II 2008 Probability and Measure: $3 . \mathrm{II} . 24 \mathrm{~J} \quad$ 

https://questions.tripos.org/part-ii/2008-97/

**Comment by stevezshen** [comment]
(i) $\to^d$ is defined where $X(t)$ is cts. 
(ii) 
Recall the formula for image measure: if $\nu(A) = \mu\left(f^{-1}(A)\right)$, then $$\int g\;\text{d}\nu = \int g\circ f\;\text{d}\mu$$. Hence, $$\mathbb{E}[X] = \int_{\Omega} X\;\text{d}\mathbb{P} = \int_{\mathbb{R}} \;\text{d}\left(\mathbb{P}\circ X^{-1}\right)$$. That is, the mean only depends on the distribution.
So now recall that $X_n\to X$ in distribution implies that $\tilde{X}_n\to\tilde{X}$ almost surely, where $\tilde{Y}$ has the same distribution as $Y$. We apply the standard theorem to say that $\mathbb{E}\left[\left|\tilde{X}_n-\tilde{X}\right|\right] \to 0$. So done.
